joomla/joomla-cms
# Update tinyMCE to 4.5.3

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/14005 .

### Summary of Changes
Usual update of tinyMCE's code
 
### Testing Instructions
Apply patch and check that everything works as expected, especially the reported bug!

### Documentation Changes Required


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/joomla/joomla-cms/14024)
